File: 프로그래머스/0/120837. 개미 군단/개미 군단.py
def solution(hp):
    a = 0
    b = 0
    c = 0
    n = a+b+c
    while hp - (5*a + 3*b + c) > 0:
        if hp - (5*a + 3*b + c) >= 5:
            a += 1
        elif hp - (5*a + 3*b + c) >= 3:
            b += 1
        else:
            c += 1
        
    n = a+b+c
    return n

File: 프로그래머스/0/120837. 개미 군단/개미 군단.java
class Solution {
    public int solution(int hp) {
        int answer = 0;
        int i = 0;
        int j = 0;
        int k = 0;
        while (hp > 0) {
            if (hp >= 5) {
                i = hp/5;
                hp -= 5 * i;
            } else if (hp >= 3) {
                j = hp / 3;
                hp -= 3 * j;
            } else {
                k = hp;
                hp = 0;
            }
        }
        answer = i + j + k;
        return answer;
    }
}